docs/integrations/microsoft-azure/azure-application-gateway.md

```diff
@@ -40,97 +40,9 @@ You must explicitly enable diagnostic settings for each Azure Application Gatewa
 
 When you configure the event hubs source or HTTP source, plan your source category to ease the querying process. A hierarchical approach allows you to make use of wildcards. For example: `Azure/ApplicationGateway/Logs`, `Azure/ApplicationGateway/Metrics`.
 
-
```

Comment:
remove Metrics collection using our [HTTP Logs and Metrics source] 
Metrics collection using our Azure Metrics Source.